src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+public class Parser {
```

Comment:
Consider writing javadoc method headers to conform to the coding standard of public methods!

Reply:
Thanks for the comment! I have included JavaDoc comments in a commit further downstream! :)